cmd/console-server/main.go

```diff
@@ -340,7 +131,11 @@ http://localhost:` + port + `/graphql
 		Name: "console_query_error_total",
 		Help: "Number of queries that have ended in error",
 	}, []string{"operationName"})
-	prometheus.MustRegister(queryTimeMetric, queryErrorCountMetric)
+	sessionCountMetric := prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "console_active_session_count",
```

Comment:
Should probably be console_active_session_total   (ref https://prometheus.io/docs/practices/naming/)